src/core/generator.jl

```diff
@@ -48,7 +48,7 @@
     can_export_beyond_nem_limit = false,
     can_curtail::Bool = false,
     macrs_option_years::Int = 0,
-    macrs_bonus_fraction::Real = 1.0,
```

Comment:
I'm not sure why we previously defaulted this to 1. It got set to 0 anyway (in cost_curve.jl) since macrs_option_years = 0.